sources/platform/management/subscription/node-pools.md

```diff
@@ -104,3 +108,12 @@ containing On-demand nodes, no additional steps are required.
 
 * To remove [BYON nodes](/platform/runtime/nodes/#byon-nodes) from node pools,
   delete individual nodes from their respective node pools. Refer to the documentation on [BYON nodes](/platform/runtime/nodes/#byon-nodes) for steps to delete nodes.
+
+### Using static IP in node pools.
+* Knowing the IP addresses of the build machines Shippable uses can be helpful when you need them to whitelist specific IP ranges so that build nodes can freely talk to machines/tools within their VPC or network.
+
+* You can use static IP in your subscription by creating a node pool with Static IP enabled or enabling Static IP in existing node pools.
+
+#### Limitations:
+* SSH debug builds are disabled on static IP enabled node pools.
```

Comment:
You will not be able to run [debug builds] on jobs that run in a node pool with static IP enabled.